app/components/start_page_banner/style.scss

```diff
@@ -0,0 +1,48 @@
+@import "../base.scss";
+@import "../../webpacker/styles/inverted";
+
+.start-page-banner {
```

Comment:
The design system suggests prefixing styles with `app-` - ([link](use-a-unique-prefix-for-component-names))